plone-and-python-community.rst

```diff
@@ -1830,13 +1829,13 @@ Plone Roadmap 2020
 
 .. container:: overlay centered
 
-    **Onboard the Plone Community and join the journey**
+    **Join the Plone Community and join the journey**
 
     See you at
 
-    * Plone Open Garden 2017 - Sorrento - Italiy - 18-22. April 2017
-    * PloneConf 2017 Barcelona - Catalonia - 16-22.  October 2017
-    * or any Sprint or Event, ...
+    * Plone Open Garden 2017 - Sorrento - Italy - April 18-22, 2017
+    * PloneConf 2017 Barcelona - Catalonia - October 16-22, 2017
+    * or any sprint or event, ... plone.org/events
```

Comment:
I will get the Slovenia sprint added to plone.org/events next in case people go there to see